llvm/lib/Transforms/Coroutines/CoroElide.cpp

```diff
@@ -33,24 +33,46 @@ static cl::opt<std::string> CoroElideInfoOutputFilename(
 
 namespace {
 // Created on demand if the coro-elide pass has work to do.
-struct Lowerer : coro::LowererBase {
+class FunctionElideManager {
```

Comment:
The name seems not good. It doesn't manage anything. Maybe we can call it `*Info`